pandas/core/ops.py

```diff
@@ -625,16 +626,21 @@ def _get_op_name(op, special):
            'desc': 'Greater than or equal to',
            'reverse': None,
            'series_examples': None}
-}
+}  # type: Dict[str, Dict[str, Optional[Union[bool, str]]]]
```

Comment:
Might be overlooking it but when is the value here a bool? If that's not required and we get rid of it can we do away with the complexity below?